src/components/Form.js

```diff
@@ -135,8 +135,34 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const m = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matches = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+
+                // add any html tags that are considered valid workspace names to ignoreRegxList
+                const ignoreRegxList = CONST.VALID_WORKSPACENAMES;
+                let isMatch = ignoreRegxList.some(r => r.test(inputValue));
```

Comment:
Do we actually need to assign the regex to separate const rather than using it in the next line directly?

Reply:
I moved the regex out of const.js and defined it directly in Form.js. The regex ignoreRegxList is also being used line 160.